views.py

```diff
@@ -654,7 +657,9 @@ def edit_annotation_base(request, image_pid, anno_pid, redirect_url):
         context_data.update(get_bound_edit_forms(annotation, AnnotationForm, PersonFormSet, InscriptionFormSet))
 
     image_link = 'https://%s/viewers/image/zoom/%s' % (BDR_SERVER, image_pid)
-    context_data.update({'image_link': image_link})
+    person_helper = PersonFormHelper()
```

Comment:
I was following this example (for formsets):
http://django-crispy-forms.readthedocs.org/en/latest/crispy_tag_formsets.html
Do you see a way to do it inline with formsets?